epersongroups.md

```diff
@@ -0,0 +1,70 @@
+# EPerson groups Endpoints
+[Back to the list of all defined endpoints](endpoints.md)
+
+## Main Endpoint
+**GET /api/eperson/groups**
+
+## Single EPerson Group
+**GET /api/eperson/groups/<:uuid>**
+
+```json
+{
+  "id": "617cf46b-535c-42d5-9d22-327ce2eff6dc",
+  "uuid": "617cf46b-535c-42d5-9d22-327ce2eff6dc",
+  "name": "Administrator",
+  "handle": null,
+  "metadata": [
+    {
+      "key": "dc.title",
+      "value": "Administrator",
+      "language": null
+    }
+  ],
+  "permanent": true,
+  "type": "group",
+  "_links": {
+    "groups": {
+      "href": "https://dspace7-internal.atmire.com/rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc/groups"
+    },
+    "self": {
+      "href": "https://dspace7-internal.atmire.com/rest/api/eperson/groups/617cf46b-535c-42d5-9d22-327ce2eff6dc"
```

Comment:
Please use the official REST & UI demo web sites for examples. This will allow us to easily replace the reference to dspace.org if needed

Reply:
This was solved